examples/SimpleRagas/SimpleRagas.ipynb

```diff
@@ -0,0 +1,1018 @@
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add the context and answer to the relevant data:\n",
+    "rag_results_w_context: List[dict] = [\n",
+    "    {**row, **result} for row, result in zip(data[:NUM_QA_PAIRS], retrieval_qa_results)\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'input': 'What is the purpose of starred docs in Coda?',\n",
+       " 'expected': 'Starring docs in Coda helps to mark documents of personal importance and organizes them in a specific section called My Shortcuts for easy access.',\n",
+       " 'metadata': {'document_id': '8179780',\n",
+       "  'section_id': 0,\n",
+       "  'question_idx': 0,\n",
+       "  'answer_idx': 0,\n",
+       "  'id': 0,\n",
+       "  'split': 'train'},\n",
+       " 'context': \"Not all Coda docs are used in the same way. You'll inevitably have a few that you use every week, and some that you'll only use once. This is where starred docs can help you stay organized.\\n\\n\\n\\nStarring docs is a great way to mark docs of personal importance. After you star a doc, it will live in a section on your doc list called **[My Shortcuts](https://coda.io/shortcuts)**. All starred docs, even from multiple different workspaces, will live in this section.\\n\\n\\n\\nStarring docs only saves them to your personal My Shortcuts. It doesn’t affect the view for others in your workspace. If you’re wanting to shortcut docs not just for yourself but also for others in your team or workspace, you’ll [use pinning](https://help.coda.io/en/articles/2865511-starred-pinned-docs) instead.\\n------\\n**Star your docs**\\n==================\\n\\n\\nTo star a doc, hover over its name in the doc list and click the star icon. Alternatively, you can star a doc from within the doc itself. Hover over the doc title in the upper left corner, and click on the star.\\n\\n\\n\\nOnce you star a doc, you can access it quickly from the [My Shortcuts](https://coda.io/shortcuts) tab of your doc list.\\n\\n\\n\\n![](https://downloads.intercomcdn.com/i/o/793964361/55a80927217f85d68d44a3c3/Star+doc+to+my+shortcuts.gif)And, as your doc needs change, simply click the star again to un-star the doc and remove it from **My Shortcuts**.\",\n",
+       " 'n_contexts': 2,\n",
+       " 'answer': 'The purpose of starred docs in Coda is to mark docs of personal importance and quickly access them from the My Shortcuts section of the doc list. Starred docs are a way to stay organized and easily find the documents that are used frequently or are of personal significance.'}"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "rag_results_w_context[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Baseline RAGAS metrics with autoeval\n",
```

Comment:
I think it'd be great if we could significantly compress the prelude to this. Maybe we pre-package the questions/answers? Can we make all the RAG implementation stuff fit in one block?